src/rhub/bare_metal/model/data/kickstart_debug_script.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# based on https://access.redhat.com/solutions/20358
+
+cd /tmp; mkdir cmd-outputs
+W() { local OutputFile=$(tr ' /' '_' <<<"$@"); $@ >cmd-outputs/$OutputFile; }
+if [[ ! -f ks.cfg ]] && [[ ! -f /run/install/ks.cfg ]]; then echo No kickstart present >ks.cfg; fi
+ls anaconda-tb-* &>/dev/null || kill -USR2 $(</var/run/anaconda.pid)
+W cat /mnt/sysimage/root/anaconda-ks.cfg
+W cat /run/install/ks.cfg
+W date
+W dmesg
+W dmidecode
+W lspci -vvnn
+W fdisk -cul
+W parted -l
+W ls -lR /dev
+W dmsetup ls --tree
+W lvm pvs
+W lvm vgs
+W lvm lvs
+W cat /proc/partitions
+W mount
+W df -h
+W cat /proc/meminfo
+W cat /proc/cpuinfo
+W ps axf
+W lsof
+W ip -s li
+W ip a
+W ip r
```

Comment:
If anaconda configured, wondering if we should gather logs that current users come to expect beneath
/var/log/anaconda. Also an important log is console.log Not sure it is included above.  without digging too much into it, I think journalctl will give us this info.